bot.py:42 #INFO     [2023-07-29 04:52:59,403] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 04:52:59,891] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 04:53:00,238] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 04:53:00,288] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 04:53:00,289] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 04:53:00,332] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:439 #WARNING  [2023-07-29 04:53:29,953] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 04:53:29,955] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 04:53:29,955] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 04:58:35,900] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 04:58:36,371] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 04:58:36,655] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 04:58:36,705] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 04:58:36,706] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 04:58:36,748] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 04:58:41,700] - aiogram.event - Update id=896828957 is not handled. Duration 0 ms by bot id=6506365433
dispatcher.py:160 #INFO     [2023-07-29 04:58:44,857] - aiogram.event - Update id=896828958 is not handled. Duration 0 ms by bot id=6506365433
dispatcher.py:439 #WARNING  [2023-07-29 04:58:50,452] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 04:58:50,453] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 04:58:50,453] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 05:01:16,491] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 05:01:16,943] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 05:01:17,240] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 05:01:17,291] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 05:01:17,292] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 05:01:17,337] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 05:01:19,737] - aiogram.event - Update id=896828959 is handled. Duration 413 ms by bot id=6506365433
dispatcher.py:160 #INFO     [2023-07-29 05:02:38,761] - aiogram.event - Update id=896828960 is handled. Duration 409 ms by bot id=6506365433
dispatcher.py:439 #WARNING  [2023-07-29 05:05:05,711] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 05:05:05,712] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 05:05:05,713] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 05:05:16,602] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 05:05:17,047] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 05:05:17,330] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 05:05:17,381] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 05:05:17,381] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 05:05:17,427] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 05:08:32,323] - aiogram.event - Update id=896828961 is handled. Duration 428 ms by bot id=6506365433
dispatcher.py:160 #INFO     [2023-07-29 05:09:49,695] - aiogram.event - Update id=896828962 is handled. Duration 407 ms by bot id=6506365433
dispatcher.py:160 #INFO     [2023-07-29 05:10:02,055] - aiogram.event - Update id=896828963 is handled. Duration 245 ms by bot id=6506365433
dispatcher.py:439 #WARNING  [2023-07-29 07:07:41,636] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:07:41,637] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:07:41,637] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:13:12,047] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:13:12,545] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:13:12,823] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:13:12,873] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:13:12,874] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:13:12,917] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:13:56,413] - aiogram.event - Update id=896828968 is handled. Duration 404 ms by bot id=6506365433
dispatcher.py:160 #INFO     [2023-07-29 07:14:14,979] - aiogram.event - Update id=896828969 is handled. Duration 528 ms by bot id=6506365433
dispatcher.py:208 #ERROR    [2023-07-29 07:15:01,461] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:01,462] - aiogram.dispatcher - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:02,502] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:02,502] - aiogram.dispatcher - Sleep for 1.365931 seconds and try again... (tryings = 1, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:03,910] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:03,911] - aiogram.dispatcher - Sleep for 1.660808 seconds and try again... (tryings = 2, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:05,618] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:05,619] - aiogram.dispatcher - Sleep for 2.094166 seconds and try again... (tryings = 3, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:07,760] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:07,760] - aiogram.dispatcher - Sleep for 2.792524 seconds and try again... (tryings = 4, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:10,600] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:10,601] - aiogram.dispatcher - Sleep for 3.677093 seconds and try again... (tryings = 5, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:14,325] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:14,325] - aiogram.dispatcher - Sleep for 4.659437 seconds and try again... (tryings = 6, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:19,030] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:19,031] - aiogram.dispatcher - Sleep for 4.913897 seconds and try again... (tryings = 7, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:23,996] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:23,997] - aiogram.dispatcher - Sleep for 5.132795 seconds and try again... (tryings = 8, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:29,178] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:29,179] - aiogram.dispatcher - Sleep for 5.039978 seconds and try again... (tryings = 9, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:34,260] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:34,261] - aiogram.dispatcher - Sleep for 5.195426 seconds and try again... (tryings = 10, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:39,504] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:39,505] - aiogram.dispatcher - Sleep for 5.134388 seconds and try again... (tryings = 11, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:44,746] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:44,746] - aiogram.dispatcher - Sleep for 5.047381 seconds and try again... (tryings = 12, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:49,840] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:49,840] - aiogram.dispatcher - Sleep for 5.070772 seconds and try again... (tryings = 13, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:54,957] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:54,958] - aiogram.dispatcher - Sleep for 4.932635 seconds and try again... (tryings = 14, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:15:59,934] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:15:59,934] - aiogram.dispatcher - Sleep for 4.915873 seconds and try again... (tryings = 15, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:04,894] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:04,894] - aiogram.dispatcher - Sleep for 5.067627 seconds and try again... (tryings = 16, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:10,012] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:10,013] - aiogram.dispatcher - Sleep for 5.058350 seconds and try again... (tryings = 17, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:15,125] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:15,125] - aiogram.dispatcher - Sleep for 5.070333 seconds and try again... (tryings = 18, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:20,246] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:20,247] - aiogram.dispatcher - Sleep for 4.969163 seconds and try again... (tryings = 19, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:25,261] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:25,261] - aiogram.dispatcher - Sleep for 5.006889 seconds and try again... (tryings = 20, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:30,313] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:30,314] - aiogram.dispatcher - Sleep for 4.965992 seconds and try again... (tryings = 21, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:35,326] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:35,327] - aiogram.dispatcher - Sleep for 5.023278 seconds and try again... (tryings = 22, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:40,397] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:40,397] - aiogram.dispatcher - Sleep for 4.869469 seconds and try again... (tryings = 23, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:45,315] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:45,316] - aiogram.dispatcher - Sleep for 5.187799 seconds and try again... (tryings = 24, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:50,548] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:50,548] - aiogram.dispatcher - Sleep for 5.238679 seconds and try again... (tryings = 25, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:16:55,842] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:16:55,842] - aiogram.dispatcher - Sleep for 5.102037 seconds and try again... (tryings = 26, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:00,990] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:00,990] - aiogram.dispatcher - Sleep for 5.022865 seconds and try again... (tryings = 27, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:06,058] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:06,059] - aiogram.dispatcher - Sleep for 5.091253 seconds and try again... (tryings = 28, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:11,193] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:11,193] - aiogram.dispatcher - Sleep for 4.935985 seconds and try again... (tryings = 29, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:16,180] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:16,180] - aiogram.dispatcher - Sleep for 5.104565 seconds and try again... (tryings = 30, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:21,334] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:21,335] - aiogram.dispatcher - Sleep for 5.129132 seconds and try again... (tryings = 31, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:26,510] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:26,511] - aiogram.dispatcher - Sleep for 4.883557 seconds and try again... (tryings = 32, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:31,443] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:31,443] - aiogram.dispatcher - Sleep for 4.919809 seconds and try again... (tryings = 33, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:36,409] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:36,410] - aiogram.dispatcher - Sleep for 5.023079 seconds and try again... (tryings = 34, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:41,479] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:41,480] - aiogram.dispatcher - Sleep for 5.086267 seconds and try again... (tryings = 35, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:46,614] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:46,614] - aiogram.dispatcher - Sleep for 5.096988 seconds and try again... (tryings = 36, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:51,758] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:51,758] - aiogram.dispatcher - Sleep for 5.074088 seconds and try again... (tryings = 37, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:17:56,875] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:17:56,875] - aiogram.dispatcher - Sleep for 4.982690 seconds and try again... (tryings = 38, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:18:01,912] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:18:01,912] - aiogram.dispatcher - Sleep for 5.008717 seconds and try again... (tryings = 39, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:18:06,970] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:18:06,970] - aiogram.dispatcher - Sleep for 5.096383 seconds and try again... (tryings = 40, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:18:12,111] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:18:12,112] - aiogram.dispatcher - Sleep for 4.997018 seconds and try again... (tryings = 41, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:18:17,156] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:18:17,156] - aiogram.dispatcher - Sleep for 4.940170 seconds and try again... (tryings = 42, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:18:22,147] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:18:22,147] - aiogram.dispatcher - Sleep for 4.993663 seconds and try again... (tryings = 43, bot id = 6506365433)
dispatcher.py:208 #ERROR    [2023-07-29 07:18:27,194] - aiogram.dispatcher - Failed to fetch updates - TelegramUnauthorizedError: Telegram server says Unauthorized
dispatcher.py:210 #WARNING  [2023-07-29 07:18:27,194] - aiogram.dispatcher - Sleep for 4.725386 seconds and try again... (tryings = 44, bot id = 6506365433)
dispatcher.py:439 #WARNING  [2023-07-29 07:18:28,939] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:18:28,940] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6506365433 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:18:28,941] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:18:36,699] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:18:40,562] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:18:41,007] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:18:41,058] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:18:41,059] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:18:41,098] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:18:48,450] - aiogram.event - Update id=61482795 is handled. Duration 462 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 07:39:28,745] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:39:28,747] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:39:28,747] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:48:39,312] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:48:39,783] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:48:40,072] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:48:40,123] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:48:40,124] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:48:40,161] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:48:47,423] - aiogram.event - Update id=61482796 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 07:48:47,423] - aiogram.event - Cause exception while process update id=61482796 by bot id=6578436252
AttributeError: 'NoneType' object has no attribute 'phone_number'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 20, in admin_start
    phone_number = m.contact.phone_number
                   ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'phone_number'
dispatcher.py:439 #WARNING  [2023-07-29 07:49:03,665] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:49:03,666] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:49:03,666] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:49:10,927] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:49:11,404] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:49:11,739] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:49:11,789] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:49:11,790] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:49:11,826] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:49:15,845] - aiogram.event - Update id=61482797 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 07:49:15,845] - aiogram.event - Cause exception while process update id=61482797 by bot id=6578436252
AttributeError: 'NoneType' object has no attribute 'phone_number'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 20, in admin_start
    phone_number = m.contact.phone_number
                   ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'phone_number'
dispatcher.py:439 #WARNING  [2023-07-29 07:51:30,762] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:51:30,763] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:51:30,763] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:51:38,576] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:51:39,027] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:51:39,306] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:51:39,356] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:51:39,359] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:51:39,397] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:51:42,365] - aiogram.event - Update id=61482798 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 07:51:42,365] - aiogram.event - Cause exception while process update id=61482798 by bot id=6578436252
AttributeError: 'NoneType' object has no attribute 'phone_number'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 20, in admin_start
    phone_number = m.contact.phone_number
                   ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'phone_number'
dispatcher.py:439 #WARNING  [2023-07-29 07:52:49,755] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:52:49,756] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:52:49,757] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:52:58,992] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:52:59,430] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:52:59,704] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:52:59,754] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:52:59,755] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:52:59,791] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:53:04,578] - aiogram.event - Update id=61482799 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 07:53:04,579] - aiogram.event - Cause exception while process update id=61482799 by bot id=6578436252
AttributeError: 'NoneType' object has no attribute 'phone_number'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 20, in admin_start
    phone_number = m.contact.phone_number
                   ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'phone_number'
dispatcher.py:439 #WARNING  [2023-07-29 07:53:19,796] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 07:53:19,797] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 07:53:19,797] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 07:53:28,342] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 07:53:28,788] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 07:53:29,071] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 07:53:29,122] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 07:53:29,123] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 07:53:29,157] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 07:53:44,245] - aiogram.event - Update id=61482800 is handled. Duration 405 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 09:17:38,686] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:17:38,691] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:17:38,692] - aiogram.dispatcher - Polling stopped
bot.py:42 #INFO     [2023-07-29 09:34:59,722] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 09:35:00,171] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 09:35:00,460] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 09:35:00,510] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 09:35:00,511] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 09:35:00,550] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 09:35:05,707] - aiogram.event - Update id=61482801 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 09:35:05,707] - aiogram.event - Cause exception while process update id=61482801 by bot id=6578436252
AttributeError: type object 'Users' has no attribute 'get_user'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 23, in admin_start
    user_data = Users.get_user(user_id)
                ^^^^^^^^^^^^^^
AttributeError: type object 'Users' has no attribute 'get_user'
dispatcher.py:439 #WARNING  [2023-07-29 09:35:13,809] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:35:13,811] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:35:13,811] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 09:37:15,775] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 09:37:16,212] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 09:37:16,485] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 09:37:16,536] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 09:37:16,536] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 09:37:16,576] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 09:37:20,157] - aiogram.event - Update id=61482802 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 09:37:20,158] - aiogram.event - Cause exception while process update id=61482802 by bot id=6578436252
AttributeError: type object 'Users' has no attribute 'get_user'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 23, in admin_start
    user_data = Users.get_user(user_id)
                ^^^^^^^^^^^^^^
AttributeError: type object 'Users' has no attribute 'get_user'
dispatcher.py:439 #WARNING  [2023-07-29 09:41:37,632] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:41:37,633] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:41:37,634] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 09:41:46,196] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 09:41:46,854] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 09:41:47,148] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 09:41:47,198] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 09:41:47,199] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 09:41:47,242] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 09:41:50,246] - aiogram.event - Update id=61482803 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 09:41:50,246] - aiogram.event - Cause exception while process update id=61482803 by bot id=6578436252
AttributeError: type object 'Users' has no attribute 'get_user'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 23, in admin_start
    user_data = Users.get_user(user_id)
                ^^^^^^^^^^^^^^
AttributeError: type object 'Users' has no attribute 'get_user'
dispatcher.py:439 #WARNING  [2023-07-29 09:46:05,329] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:46:05,330] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:46:05,331] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 09:47:48,340] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 09:47:48,828] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 09:47:49,110] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 09:47:49,161] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 09:47:49,161] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 09:47:49,201] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 09:47:53,718] - aiogram.event - Update id=61482804 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 09:47:53,719] - aiogram.event - Cause exception while process update id=61482804 by bot id=6578436252
AttributeError: type object 'Users' has no attribute 'get_user'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 23, in admin_start
    user_data = Users.get_user(user_id)
                ^^^^^^^^^^^^^^
AttributeError: type object 'Users' has no attribute 'get_user'
dispatcher.py:160 #INFO     [2023-07-29 09:47:58,175] - aiogram.event - Update id=61482805 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 09:47:58,175] - aiogram.event - Cause exception while process update id=61482805 by bot id=6578436252
AttributeError: type object 'Users' has no attribute 'get_user'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 23, in admin_start
    user_data = Users.get_user(user_id)
                ^^^^^^^^^^^^^^
AttributeError: type object 'Users' has no attribute 'get_user'
dispatcher.py:439 #WARNING  [2023-07-29 09:48:00,117] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:48:00,118] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:48:00,118] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 09:49:58,723] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 09:49:59,173] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 09:49:59,475] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 09:49:59,526] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 09:49:59,526] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 09:49:59,563] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-29 09:50:04,880] - aiogram.event - Update id=61482806 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-29 09:50:04,880] - aiogram.event - Cause exception while process update id=61482806 by bot id=6578436252
AttributeError: type object 'Users' has no attribute 'get_user'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/handlers/admin.py", line 23, in admin_start
    user_data = Users.get_user(user_id)
                ^^^^^^^^^^^^^^
AttributeError: type object 'Users' has no attribute 'get_user'
dispatcher.py:439 #WARNING  [2023-07-29 09:51:35,702] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:51:35,703] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:51:35,703] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 09:51:43,246] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 09:51:43,683] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 09:51:43,953] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 09:51:44,003] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 09:51:44,004] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 09:51:44,042] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:25 #INFO     [2023-07-29 09:51:46,956] - root - None
db.py:30 #INFO     [2023-07-29 09:51:46,956] - root - Trying to save user.
db.py:35 #INFO     [2023-07-29 09:51:46,961] - root - User successfully saved!
admin.py:40 #INFO     [2023-07-29 09:51:47,274] - root - ManagerImpl.start() missing 1 required positional argument: 'state'
dispatcher.py:160 #INFO     [2023-07-29 09:51:47,275] - aiogram.event - Update id=61482807 is handled. Duration 328 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 09:52:34,039] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 09:52:34,040] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 09:52:34,040] - aiogram.dispatcher - Polling stopped
bot.py:39 #INFO     [2023-07-29 10:05:00,294] - __main__ - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:05:00,767] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:05:01,067] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:05:01,117] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:05:01,118] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:05:01,158] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:26 #INFO     [2023-07-29 10:05:04,487] - tgbot.handlers.admin - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:41 #INFO     [2023-07-29 10:05:05,040] - tgbot.handlers.admin - ManagerImpl.start() missing 1 required positional argument: 'state'
dispatcher.py:160 #INFO     [2023-07-29 10:05:05,041] - aiogram.event - Update id=61482808 is handled. Duration 559 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 10:06:35,612] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 10:06:35,613] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 10:06:35,613] - aiogram.dispatcher - Polling stopped
bot.py:39 #INFO     [2023-07-29 10:07:37,446] - __main__ - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:07:37,850] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:07:38,189] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:07:38,239] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:07:38,240] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:07:38,280] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:26 #INFO     [2023-07-29 10:07:42,273] - tgbot.handlers.admin - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:27 #INFO     [2023-07-29 10:07:42,274] - tgbot.handlers.admin - Hello admin!
admin.py:42 #INFO     [2023-07-29 10:07:42,582] - tgbot.handlers.admin - ManagerImpl.start() missing 1 required positional argument: 'state'
dispatcher.py:160 #INFO     [2023-07-29 10:07:42,582] - aiogram.event - Update id=61482809 is handled. Duration 313 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 10:09:08,891] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 10:09:08,892] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 10:09:08,893] - aiogram.dispatcher - Polling stopped
bot.py:40 #INFO     [2023-07-29 10:09:17,146] - __main__ - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:09:17,618] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:09:17,896] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:09:17,946] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:09:17,947] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:09:17,984] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:26 #INFO     [2023-07-29 10:09:21,857] - tgbot.handlers.admin - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:27 #INFO     [2023-07-29 10:09:21,857] - tgbot.handlers.admin - Hello admin!
admin.py:42 #INFO     [2023-07-29 10:09:22,165] - tgbot.handlers.admin - ManagerImpl.start() missing 1 required positional argument: 'state'
dispatcher.py:160 #INFO     [2023-07-29 10:09:22,165] - aiogram.event - Update id=61482810 is handled. Duration 313 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 10:31:14,384] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 10:31:14,386] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 10:31:14,386] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 10:33:10,458] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:33:11,097] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:33:11,379] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:33:11,430] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:33:11,430] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:33:11,470] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 10:33:15,701] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:39 #INFO     [2023-07-29 10:33:16,021] - root - ManagerImpl.start() missing 1 required positional argument: 'state'
dispatcher.py:160 #INFO     [2023-07-29 10:33:16,021] - aiogram.event - Update id=61482811 is handled. Duration 325 ms by bot id=6578436252
admin.py:24 #INFO     [2023-07-29 10:34:11,925] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:39 #INFO     [2023-07-29 10:34:12,315] - root - ManagerImpl.start() missing 1 required positional argument: 'state'
dispatcher.py:160 #INFO     [2023-07-29 10:34:12,316] - aiogram.event - Update id=61482812 is handled. Duration 391 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 10:35:08,881] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 10:35:08,882] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 10:35:08,882] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 10:35:16,143] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:35:16,600] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:35:16,880] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:35:16,931] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:35:16,932] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:35:16,965] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 10:35:18,079] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 10:35:18,646] - aiogram.event - Update id=61482813 is handled. Duration 577 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 10:55:18,743] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 10:55:18,744] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 10:55:18,745] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 10:55:27,828] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:55:28,296] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:55:28,588] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:55:28,638] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:55:28,639] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:55:28,675] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 10:55:31,753] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:40 #INFO     [2023-07-29 10:55:32,120] - root - 'links_info'
dispatcher.py:160 #INFO     [2023-07-29 10:55:32,121] - aiogram.event - Update id=61482814 is handled. Duration 371 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 10:57:33,344] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 10:57:33,346] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 10:57:33,346] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 10:57:40,534] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 10:57:40,980] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 10:57:41,276] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 10:57:41,327] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 10:57:41,328] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 10:57:41,365] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 10:57:43,994] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:40 #INFO     [2023-07-29 10:57:44,323] - root - 'links_info'
dispatcher.py:160 #INFO     [2023-07-29 10:57:44,324] - aiogram.event - Update id=61482815 is handled. Duration 333 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 11:03:36,516] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 11:03:36,517] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 11:03:36,518] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 11:03:44,727] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 11:03:45,187] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 11:03:45,502] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 11:03:45,553] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 11:03:45,554] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 11:03:45,589] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 11:03:50,425] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 11:03:51,015] - aiogram.event - Update id=61482816 is handled. Duration 593 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 11:03:58,880] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 11:03:58,881] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 11:03:58,882] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 11:04:07,315] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 11:04:07,741] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 11:04:08,016] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 11:04:08,066] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 11:04:08,067] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 11:04:08,103] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 11:04:10,484] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 11:04:11,078] - aiogram.event - Update id=61482817 is handled. Duration 599 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 11:05:13,299] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 11:05:13,300] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 11:05:13,300] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 11:05:20,579] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 11:05:21,090] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 11:05:21,372] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 11:05:21,422] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 11:05:21,423] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 11:05:21,463] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 11:05:25,850] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
admin.py:40 #INFO     [2023-07-29 11:05:26,184] - root - 'NoneType' object is not iterable
dispatcher.py:160 #INFO     [2023-07-29 11:05:26,184] - aiogram.event - Update id=61482818 is handled. Duration 345 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 11:05:57,690] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 11:05:57,691] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 11:05:57,691] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 11:06:04,677] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 11:06:05,142] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 11:06:05,411] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 11:06:05,461] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 11:06:05,462] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 11:06:05,502] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 11:06:07,068] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 11:06:07,721] - aiogram.event - Update id=61482819 is handled. Duration 660 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 12:31:02,659] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 12:31:02,660] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 12:31:02,661] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 12:31:12,475] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 12:31:12,887] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 12:31:13,169] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 12:31:13,220] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 12:31:13,220] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 12:31:13,278] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 12:31:14,803] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 12:31:15,352] - aiogram.event - Update id=61482820 is handled. Duration 555 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 12:32:37,712] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 12:32:37,713] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 12:32:37,713] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 12:32:45,106] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 12:32:45,707] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 12:32:46,088] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 12:32:46,139] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 12:32:46,139] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 12:32:46,178] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 12:32:47,933] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 12:32:48,476] - aiogram.event - Update id=61482821 is handled. Duration 546 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 12:33:00,457] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 12:33:00,458] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 12:33:00,459] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-29 12:33:07,932] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-29 12:33:08,349] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-29 12:33:08,632] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-29 12:33:08,683] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-29 12:33:08,686] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-29 12:33:08,721] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-29 12:33:10,251] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 29, 9, 51, 46, 947415)]
dispatcher.py:160 #INFO     [2023-07-29 12:33:10,922] - aiogram.event - Update id=61482822 is handled. Duration 676 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-29 12:34:06,198] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-29 12:34:06,199] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-29 12:34:06,199] - aiogram.dispatcher - Polling stopped
bot.py:43 #INFO     [2023-07-30 09:06:43,034] - root - Starting bot!
bot.py:43 #INFO     [2023-07-30 09:06:59,574] - root - Starting bot!
bot.py:43 #INFO     [2023-07-30 09:07:01,173] - root - Starting bot!
bot.py:43 #INFO     [2023-07-30 09:08:15,395] - root - Starting bot!
bot.py:43 #INFO     [2023-07-30 09:08:17,074] - root - Starting bot!
bot.py:44 #INFO     [2023-07-30 09:08:26,906] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:08:27,375] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:08:27,656] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:08:27,706] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:08:27,707] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:08:27,755] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 09:08:31,435] - root - None
db.py:30 #INFO     [2023-07-30 09:08:31,436] - root - Trying to save user.
db.py:35 #INFO     [2023-07-30 09:08:31,440] - root - User successfully saved!
dispatcher.py:160 #INFO     [2023-07-30 09:08:32,353] - aiogram.event - Update id=61482827 is handled. Duration 921 ms by bot id=6578436252
dispatcher.py:160 #INFO     [2023-07-30 09:08:39,284] - aiogram.event - Update id=61482828 is handled. Duration 517 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 09:09:01,810] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 09:09:01,811] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 09:09:01,811] - aiogram.dispatcher - Polling stopped
bot.py:44 #INFO     [2023-07-30 09:16:34,329] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:16:34,847] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:16:35,183] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:16:35,233] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:16:35,234] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:16:35,271] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 09:16:37,164] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:50 #INFO     [2023-07-30 09:16:37,498] - root - 1 validation error for InlineKeyboardMarkup
inline_keyboard
  field required (type=value_error.missing)
dispatcher.py:160 #INFO     [2023-07-30 09:16:37,498] - aiogram.event - Update id=61482829 is handled. Duration 337 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 09:28:00,976] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 09:28:00,977] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 09:28:00,978] - aiogram.dispatcher - Polling stopped
bot.py:44 #INFO     [2023-07-30 09:28:09,478] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:28:09,930] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:28:10,208] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:28:10,259] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:28:10,259] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:28:10,296] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:25 #INFO     [2023-07-30 09:28:12,531] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:49 #INFO     [2023-07-30 09:28:13,145] - root - Telegram server says Bad Request: query is too old and response timeout expired or query ID is invalid
dispatcher.py:160 #INFO     [2023-07-30 09:28:13,145] - aiogram.event - Update id=61482830 is handled. Duration 617 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 09:32:08,795] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 09:32:08,796] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 09:32:08,797] - aiogram.dispatcher - Polling stopped
bot.py:44 #INFO     [2023-07-30 09:33:11,264] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:33:11,685] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:33:11,962] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:33:12,013] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:33:12,014] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:33:12,051] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-30 09:33:14,031] - aiogram.event - Update id=61482831 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-30 09:33:14,031] - aiogram.event - Cause exception while process update id=61482831 by bot id=6578436252
TypeError: admin_start() missing 1 required positional argument: 'c'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
                 ^^^^^^^^^
TypeError: admin_start() missing 1 required positional argument: 'c'
dispatcher.py:439 #WARNING  [2023-07-30 09:33:23,474] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 09:33:23,475] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 09:33:23,475] - aiogram.dispatcher - Polling stopped
bot.py:44 #INFO     [2023-07-30 09:36:55,347] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:36:55,778] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:36:56,067] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:36:56,118] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:36:56,119] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:36:56,154] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-30 09:36:58,423] - aiogram.event - Update id=61482832 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-30 09:36:58,423] - aiogram.event - Cause exception while process update id=61482832 by bot id=6578436252
TypeError: admin_start() missing 1 required positional argument: 'c'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
                 ^^^^^^^^^
TypeError: admin_start() missing 1 required positional argument: 'c'
dispatcher.py:439 #WARNING  [2023-07-30 09:37:02,854] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 09:37:02,855] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 09:37:02,855] - aiogram.dispatcher - Polling stopped
bot.py:44 #INFO     [2023-07-30 09:41:49,244] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:41:49,700] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:41:49,986] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:41:50,036] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:41:50,037] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:41:50,080] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:160 #INFO     [2023-07-30 09:41:52,877] - aiogram.event - Update id=61482833 is not handled. Duration 0 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-30 09:41:52,877] - aiogram.event - Cause exception while process update id=61482833 by bot id=6578436252
TypeError: admin_start() missing 1 required positional argument: 'c'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
                 ^^^^^^^^^
TypeError: admin_start() missing 1 required positional argument: 'c'
dispatcher.py:439 #WARNING  [2023-07-30 09:41:58,287] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 09:41:58,288] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 09:41:58,289] - aiogram.dispatcher - Polling stopped
bot.py:44 #INFO     [2023-07-30 09:42:21,510] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 09:42:21,927] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 09:42:22,249] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 09:42:22,300] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 09:42:22,300] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 09:42:22,341] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 09:42:24,001] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
dispatcher.py:160 #INFO     [2023-07-30 09:42:24,667] - aiogram.event - Update id=61482834 is handled. Duration 669 ms by bot id=6578436252
dispatcher.py:160 #INFO     [2023-07-30 09:42:32,490] - aiogram.event - Update id=61482835 is handled. Duration 440 ms by bot id=6578436252
dispatcher.py:160 #INFO     [2023-07-30 09:42:38,252] - aiogram.event - Update id=61482836 is handled. Duration 418 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 10:48:59,659] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 10:48:59,661] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 10:48:59,662] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 13:29:37,660] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 13:29:38,116] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 13:29:38,402] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 13:29:38,453] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 13:29:38,454] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 13:29:38,492] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:439 #WARNING  [2023-07-30 13:29:39,743] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 13:29:39,744] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 13:29:39,745] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 13:30:06,750] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 13:30:07,218] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 13:30:07,510] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 13:30:07,561] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 13:30:07,562] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 13:30:07,610] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 13:34:12,163] - root - None
db.py:30 #INFO     [2023-07-30 13:34:12,163] - root - Trying to save user.
db.py:35 #INFO     [2023-07-30 13:34:12,170] - root - User successfully saved!
dispatcher.py:160 #INFO     [2023-07-30 13:34:12,810] - aiogram.event - Update id=61482837 is handled. Duration 655 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 13:45:51,953] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 13:45:51,955] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 13:45:51,955] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 13:46:01,408] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 13:46:01,860] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 13:46:02,220] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 13:46:02,270] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 13:46:02,271] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 13:46:02,312] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 13:46:05,368] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
dispatcher.py:160 #INFO     [2023-07-30 13:46:05,938] - aiogram.event - Update id=61482838 is handled. Duration 575 ms by bot id=6578436252
dispatcher.py:160 #INFO     [2023-07-30 14:43:14,458] - aiogram.event - Update id=61482839 is handled. Duration 455 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 15:19:53,408] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 15:19:53,411] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 15:19:53,411] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 18:41:41,001] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 18:41:41,455] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 18:41:41,763] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 18:41:41,813] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 18:41:41,814] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 18:41:41,867] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 18:42:05,214] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
dispatcher.py:160 #INFO     [2023-07-30 18:42:05,811] - aiogram.event - Update id=61482840 is handled. Duration 602 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 18:45:19,594] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 18:45:19,605] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 18:45:19,606] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 18:45:27,061] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 18:45:27,464] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 18:45:27,770] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 18:45:27,820] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 18:45:27,821] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 18:45:27,862] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 18:45:29,098] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
dispatcher.py:160 #INFO     [2023-07-30 18:45:29,694] - aiogram.event - Update id=61482841 is handled. Duration 600 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 18:55:03,829] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 18:55:03,831] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 18:55:03,832] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 18:55:31,023] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 18:55:31,497] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 18:55:31,790] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 18:55:31,848] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 18:55:31,848] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 18:55:31,886] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 18:55:36,384] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
admin.py:40 #INFO     [2023-07-30 18:55:36,789] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 18:55:36,789] - aiogram.event - Update id=61482842 is handled. Duration 408 ms by bot id=6578436252
admin.py:24 #INFO     [2023-07-30 18:55:40,747] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
admin.py:40 #INFO     [2023-07-30 18:55:41,016] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 18:55:41,016] - aiogram.event - Update id=61482843 is handled. Duration 270 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 18:55:46,679] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 18:55:46,680] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 18:55:46,681] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:06:16,270] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:06:16,720] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:06:17,002] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:06:17,053] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:06:17,054] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:06:17,099] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:06:23,416] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:06:23,735] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:06:23,736] - aiogram.event - Update id=61482844 is handled. Duration 322 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:07:55,613] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:07:55,614] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:07:55,614] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:08:02,846] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:08:03,303] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:08:03,595] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:08:03,646] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:08:03,647] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:08:03,686] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:09:01,635] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
admin.py:40 #INFO     [2023-07-30 19:09:02,026] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:09:02,027] - aiogram.event - Update id=61482845 is handled. Duration 399 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:09:04,098] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:09:04,099] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:09:04,100] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:10:00,074] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:10:00,515] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:10:00,823] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:10:00,874] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:10:00,874] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:10:00,912] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:10:05,926] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:10:06,250] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:10:06,250] - aiogram.event - Update id=61482846 is handled. Duration 327 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:10:08,337] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:10:08,338] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:10:08,338] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:11:50,698] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:11:51,114] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:11:51,402] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:11:51,452] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:11:51,453] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:11:51,491] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:12:08,102] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:12:08,486] - root - Telegram server says Bad Request: message text is empty
dispatcher.py:160 #INFO     [2023-07-30 19:12:08,487] - aiogram.event - Update id=61482847 is handled. Duration 389 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:13:22,184] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:13:22,185] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:13:22,185] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:13:32,493] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:13:32,917] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:13:33,202] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:13:33,253] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:13:33,253] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:13:33,291] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:13:35,342] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
dispatcher.py:160 #INFO     [2023-07-30 19:13:35,922] - aiogram.event - Update id=61482848 is handled. Duration 583 ms by bot id=6578436252
admin.py:24 #INFO     [2023-07-30 19:13:46,624] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
dispatcher.py:160 #INFO     [2023-07-30 19:13:47,118] - aiogram.event - Update id=61482849 is handled. Duration 495 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:14:45,684] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:14:45,685] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:14:45,685] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:14:53,335] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:14:53,805] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:14:54,082] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:14:54,132] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:14:54,133] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:14:54,171] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:14:59,231] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
admin.py:40 #INFO     [2023-07-30 19:14:59,564] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:14:59,565] - aiogram.event - Update id=61482850 is handled. Duration 337 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:16:06,714] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:16:06,715] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:16:06,715] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:16:14,127] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:16:14,542] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:16:14,821] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:16:14,872] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:16:14,872] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:16:14,911] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:16:17,817] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:16:18,133] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:16:18,133] - aiogram.event - Update id=61482851 is handled. Duration 319 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:16:27,661] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:16:27,661] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:16:27,662] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:16:35,219] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:16:35,636] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:16:35,920] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:16:35,970] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:16:35,971] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:16:36,018] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:16:37,760] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
dispatcher.py:160 #INFO     [2023-07-30 19:16:38,312] - aiogram.event - Update id=61482852 is handled. Duration 556 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:17:24,819] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:17:24,820] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:17:24,820] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:17:32,629] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:17:33,052] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:17:33,339] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:17:33,389] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:17:33,390] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:17:33,433] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:17:54,898] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
admin.py:40 #INFO     [2023-07-30 19:17:55,243] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:17:55,243] - aiogram.event - Update id=61482853 is handled. Duration 349 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:18:46,528] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:18:46,529] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:18:46,530] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:18:53,609] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:18:54,031] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:18:54,328] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:18:54,378] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:18:54,379] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:18:54,416] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:18:57,612] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:18:57,933] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:18:57,933] - aiogram.event - Update id=61482854 is handled. Duration 324 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:20:30,203] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:20:30,204] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:20:30,205] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:20:37,125] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:20:37,566] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:20:37,845] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:20:37,895] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:20:37,896] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:20:37,935] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:20:41,003] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:20:41,334] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:20:41,334] - aiogram.event - Update id=61482855 is handled. Duration 334 ms by bot id=6578436252
dispatcher.py:160 #INFO     [2023-07-30 19:21:02,412] - aiogram.event - Update id=61482856 is handled. Duration 338 ms by bot id=6578436252
dispatcher.py:160 #INFO     [2023-07-30 19:22:09,962] - aiogram.event - Update id=61482857 is not handled. Duration 1 ms by bot id=6578436252
dispatcher.py:303 #ERROR    [2023-07-30 19:22:09,963] - aiogram.event - Cause exception while process update id=61482857 by bot id=6578436252
AttributeError: 'Dispatcher' object has no attribute 'get_media_id'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 297, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 146, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 264, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 120, in process_message
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/app/VVSPIN_BOT/tgbot/middlewares/config.py", line 18, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 131, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 115, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 111, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/router.py", line 124, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/telegram.py", line 123, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager_middleware.py", line 44, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/context/intent_middleware.py", line 167, in context_saver_middleware
    result = await handler(event, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram/dispatcher/event/handler.py", line 42, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/dialog.py", line 132, in _message_handler
    await dialog_manager.show()
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager.py", line 274, in show
    await self._fix_cached_media_id(new_message)
  File "/usr/local/lib/python3.11/site-packages/aiogram_dialog/manager/manager.py", line 296, in _fix_cached_media_id
    new_message.media.file_id = await self.media_id_storage.get_media_id(
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:439 #WARNING  [2023-07-30 19:23:36,287] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:23:36,288] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:23:36,288] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:23:44,759] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:23:45,221] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:23:45,505] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:23:45,556] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:23:45,556] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:23:45,596] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:23:49,110] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:23:49,426] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:23:49,427] - aiogram.event - Update id=61482858 is handled. Duration 319 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:23:51,234] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:23:51,235] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:23:51,235] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:43:13,333] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:43:13,759] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:43:14,033] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:43:14,084] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:43:14,085] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:43:14,123] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:43:19,694] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:43:20,012] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:43:20,012] - aiogram.event - Update id=61482859 is handled. Duration 322 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:44:59,303] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:44:59,305] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:44:59,305] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:45:06,689] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:45:07,128] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:45:07,422] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:45:07,473] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:45:07,474] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:45:07,518] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:45:10,292] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:45:10,613] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:45:10,614] - aiogram.event - Update id=61482860 is handled. Duration 325 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:45:47,725] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:45:47,726] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:45:47,726] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:45:54,558] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:45:54,966] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:45:55,244] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:45:55,295] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:45:55,296] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:45:55,343] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:45:58,358] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:45:58,674] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:45:58,674] - aiogram.event - Update id=61482861 is handled. Duration 320 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:47:26,903] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:47:26,904] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:47:26,905] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:47:34,698] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:47:35,126] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:47:35,409] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:47:35,460] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:47:35,461] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:47:35,498] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:47:38,182] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:47:38,531] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:47:38,531] - aiogram.event - Update id=61482862 is handled. Duration 353 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:48:27,794] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:48:27,795] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:48:27,795] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:48:34,947] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:48:35,353] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:48:35,623] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:48:35,673] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:48:35,674] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:48:35,711] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:48:37,148] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 19:48:37,458] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 19:48:37,459] - aiogram.event - Update id=61482863 is handled. Duration 313 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 19:50:34,486] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 19:50:34,487] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 19:50:34,488] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 19:50:41,662] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 19:50:42,088] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 19:50:42,359] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 19:50:42,410] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 19:50:42,411] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 19:50:42,450] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 19:50:45,063] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
dispatcher.py:160 #INFO     [2023-07-30 19:50:45,590] - aiogram.event - Update id=61482864 is handled. Duration 530 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:00:16,384] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:00:16,386] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:00:16,386] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:00:48,212] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:00:48,677] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:00:48,962] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:00:49,012] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:00:49,013] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:00:49,053] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:01:00,198] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 20:01:00,533] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:01:00,533] - aiogram.event - Update id=61482865 is handled. Duration 337 ms by bot id=6578436252
admin.py:24 #INFO     [2023-07-30 20:01:38,666] - root - [6188512171, 'vvspins', None, None, datetime.datetime(2023, 7, 30, 13, 34, 12, 155929)]
admin.py:40 #INFO     [2023-07-30 20:01:38,994] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:01:38,995] - aiogram.event - Update id=61482866 is handled. Duration 330 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:01:40,613] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:01:40,614] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:01:40,614] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:02:26,485] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:02:26,900] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:02:27,200] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:02:27,251] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:02:27,252] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:02:27,292] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:02:30,751] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 20:02:31,066] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:02:31,067] - aiogram.event - Update id=61482867 is handled. Duration 319 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:06:29,173] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:06:29,175] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:06:29,175] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:06:37,661] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:06:38,126] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:06:38,405] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:06:38,455] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:06:38,456] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:06:38,500] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:06:39,692] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 20:06:40,015] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:06:40,015] - aiogram.event - Update id=61482868 is handled. Duration 326 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:09:23,011] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:09:23,012] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:09:23,013] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:09:29,920] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:09:30,355] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:09:30,652] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:09:30,702] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:09:30,703] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:09:30,742] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:09:33,203] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
dispatcher.py:160 #INFO     [2023-07-30 20:09:33,790] - aiogram.event - Update id=61482869 is handled. Duration 590 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:15:06,633] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:15:06,634] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:15:06,634] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:15:14,636] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:15:15,093] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:15:15,391] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:15:15,442] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:15:15,443] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:15:15,478] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:15:16,864] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 9, 8, 31, 432631)]
admin.py:40 #INFO     [2023-07-30 20:15:17,188] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:15:17,188] - aiogram.event - Update id=61482870 is handled. Duration 327 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:16:00,409] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:16:00,410] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:16:00,410] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:16:34,800] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:16:35,243] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:16:35,543] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:16:35,594] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:16:35,595] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:16:35,634] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:16:40,089] - root - None
db.py:30 #INFO     [2023-07-30 20:16:40,090] - root - Trying to save user.
db.py:35 #INFO     [2023-07-30 20:16:40,096] - root - User successfully saved!
admin.py:40 #INFO     [2023-07-30 20:16:40,437] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:16:40,437] - aiogram.event - Update id=61482871 is handled. Duration 351 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:16:42,151] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:16:42,152] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:16:42,152] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:19:29,694] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:19:30,144] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:19:30,427] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:19:30,478] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:19:30,479] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:19:30,516] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:19:34,328] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
admin.py:40 #INFO     [2023-07-30 20:19:34,641] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:19:34,642] - aiogram.event - Update id=61482872 is handled. Duration 317 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:19:36,575] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:19:36,576] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:19:36,576] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:19:59,863] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:20:00,260] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:20:00,556] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:20:00,607] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:20:00,608] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:20:00,644] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:20:05,361] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
admin.py:40 #INFO     [2023-07-30 20:20:05,679] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:20:05,680] - aiogram.event - Update id=61482873 is handled. Duration 322 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:26:15,248] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:26:15,249] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:26:15,250] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:26:25,201] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:26:25,641] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:26:25,933] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:26:25,984] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:26:25,984] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:26:26,020] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:26:28,701] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
admin.py:40 #INFO     [2023-07-30 20:26:29,022] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:26:29,023] - aiogram.event - Update id=61482874 is handled. Duration 324 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:28:13,090] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:28:13,091] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:28:13,092] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:31:39,417] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:31:40,000] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:31:40,279] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:31:40,329] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:31:40,330] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:31:40,373] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:31:47,535] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
admin.py:40 #INFO     [2023-07-30 20:31:47,849] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:31:47,849] - aiogram.event - Update id=61482875 is handled. Duration 317 ms by bot id=6578436252
admin.py:24 #INFO     [2023-07-30 20:32:01,492] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
admin.py:40 #INFO     [2023-07-30 20:32:01,733] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:32:01,733] - aiogram.event - Update id=61482876 is handled. Duration 242 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:32:11,816] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:32:11,817] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:32:11,817] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:32:19,194] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:32:19,630] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:32:19,917] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:32:19,968] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:32:19,968] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:32:20,006] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:32:20,840] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
dispatcher.py:160 #INFO     [2023-07-30 20:32:21,431] - aiogram.event - Update id=61482877 is handled. Duration 595 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:34:51,358] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:34:51,359] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:34:51,359] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:34:58,978] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:34:59,402] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:34:59,680] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:34:59,731] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:34:59,732] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:34:59,768] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
admin.py:24 #INFO     [2023-07-30 20:35:02,560] - root - [683497406, 'dankondankon', None, None, datetime.datetime(2023, 7, 30, 20, 16, 40, 86549)]
admin.py:40 #INFO     [2023-07-30 20:35:02,864] - root - 'Dispatcher' object has no attribute 'get_media_id'
dispatcher.py:160 #INFO     [2023-07-30 20:35:02,865] - aiogram.event - Update id=61482878 is handled. Duration 308 ms by bot id=6578436252
dispatcher.py:439 #WARNING  [2023-07-30 20:35:09,097] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:35:09,098] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:35:09,099] - aiogram.dispatcher - Polling stopped
bot.py:45 #INFO     [2023-07-30 20:37:59,431] - root - Starting bot!
broadcaster.py:20 #INFO     [2023-07-30 20:37:59,927] - root - Target [ID:6188512171]: success
broadcaster.py:20 #INFO     [2023-07-30 20:38:00,206] - root - Target [ID:683497406]: success
broadcaster.py:37 #INFO     [2023-07-30 20:38:00,257] - root - 2 messages successful sent.
dispatcher.py:493 #INFO     [2023-07-30 20:38:00,258] - aiogram.dispatcher - Start polling
dispatcher.py:329 #INFO     [2023-07-30 20:38:00,295] - aiogram.dispatcher - Run polling for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:439 #WARNING  [2023-07-30 20:42:57,433] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:345 #INFO     [2023-07-30 20:42:57,435] - aiogram.dispatcher - Polling stopped for bot @vvspin_bot id=6578436252 - 'VVSPIN_BOT'
dispatcher.py:520 #INFO     [2023-07-30 20:42:57,435] - aiogram.dispatcher - Polling stopped
